sdk/communication/azure-communication-networktraversal/CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Release History
+
+## 1.0.0b1 (unreleased)
```

Comment:
This doesn't match the other version forms - also this shouldn't be `unreleased`, it should be the intended release date

Reply:
Changed it to be the same as we have on Python, I asked @ankitarorabit about which one to use. Makes sense to me to use the same.